src/main/webapp/features/screens/game/view.html

```diff
@@ -1,48 +1,3 @@
-<div class="row">
+<ct-game-room ng-if="game.isNew()" game="game"></ct-game-room>
```

Comment:
but we should think about strict flow to follow it and create appropriate story, so we can leave it as is.